### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by KevinEtienne**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 
 install_requires = (
-    'djangorestframework>=2.4.3,<3',
-    'incuna_mail>=2.0.0,<3',
+    'djangorestframework>=2.4.4,<3',
+    'incuna_mail>=3.0.0,<3.1.0',
```
Comment (Ian-Foote) [inline setup.py]: Do we need to drop support for `v2`?

Comment (KevinEtienne) [inline setup.py]: I suppose we can keep `v2.0` but the difference between the two versions is making `template_name` mandatory.

Comment (KevinEtienne) [inline setup.py]: Reverting it after discussion IRL, probably best to keep support to `2.0.0`

File: setup.py [hunk 2]
```diff
@@ -5,8 +5,8 @@
 
 
 install_requires = (
-    'djangorestframework>=2.4.3,<3',
-    'incuna_mail>=2.0.0,<3',
+    'djangorestframework>=2.4.4,<3',
+    'incuna_mail>=2.0.0,<=3.0.0',
```
Comment (meshy) [inline setup.py]: Perhaps this should be `<4`?

Comment (meshy) [inline setup.py]: does it still work with version 2?

Comment (KevinEtienne) [inline setup.py]: Yes it works with v2.0 and v3.0 and maybe not v4 but v3.1?

Comment (meshy) [inline setup.py]: Well v3.1 should be backwards compatible, so if it works with v3, it should work with `<4`

Cool that it works with v2 as well

Comment (KevinEtienne) [inline setup.py]: :+1:

**Pull request merged**